monkey/monkey_island/cc/ui/src/components/pages/ConfigurePage.js

```diff
@@ -291,10 +300,12 @@ class ConfigurePageComponent extends AuthComponent {
 
   exportConfig = () => {
     this.updateConfigSection();
-    const configAsJson = JSON.stringify(this.state.configuration, null, 2);
-    const configAsBinary = new Blob([configAsJson], {type: 'text/plain;charset=utf-8'});
+    let allConfig = this.state.configuration;
+    allConfig.attack = this.state.attackConfig;
+    let allConfigAsJson = JSON.stringify(allConfig, null, 2);
+    const allConfigAsBinary = new Blob([allConfigAsJson], {type: 'text/plain;charset=utf-8'});
```

Comment:
You can probably rename `allConfig` to just `config` and change `.attack` to `.attack_matrix_ui_state`.